docs/pages/guides/deep-linking.mdx

```diff
@@ -149,50 +219,43 @@ Implementing deep links on Android (without a custom URL scheme) is somewhat sim
 }
 ```
 
-It may be desirable for links to your domain to always open your app (without presenting the user a dialog where they can choose the browser or a different handler). You can implement this with Android App Links, which use a similar verification process as Universal Links on iOS.
+### Android Assetlinks
 
-<Step label="1">
-  You must publish a JSON file at `/.well-known/assetlinks.json` specifying your app ID and which
-  links should be opened by your app. You need to format this file that includes `package_name` (app
-  ID) and `sha256_cert_fingerprints`. You can get these values from your app's [Google Play
-  Console](https://play.google.com/console/) account and then go to **Release** > **Setup** > **App
-  integrity** > **App Signing**. Under **Digital Asset Links JSON**, you will see a list of
-  fingerprints for your app. Copy and paste the JSON into the **assetlinks.json**. For more
-  information, see [Android's
-  documentation](https://developer.android.com/training/app-links/verify-android-applinks#web-assoc).
-</Step>
-
-<Step label="2">
-  Add `"autoVerify": true` to the intent filter in your [app config](/workflow/configuration/). This
-  tells Android to check for your **assetlinks.json** on your server and register your app as the
-  automatic handler for the specified paths:
-</Step>
+It may be desirable for links to your domain to always open your app (without presenting the user a dialog where they can choose the browser or a different handler). You can implement this with Android App Links, which use a similar verification process as Universal Links on iOS.
 
-```json app.json
-{
-  "expo": {
-    "android": {
-      "intentFilters": [
-        {
-          "action": "VIEW",
-          /* @info Add autoVerify property to the intent filter in app config.*/
-          "autoVerify": true,
-          /* @end */
-          "data": [
-            {
-              "scheme": "https",
-              "host": "*.myapp.io",
-              "pathPrefix": "/records"
-            }
-          ],
-          "category": ["BROWSABLE", "DEFAULT"]
-        }
+Create a JSON file for the website verification (aka [digital asset links](https://developers.google.com/digital-asset-links/v1/getting-started) file) at `public/.well-known/assetlinks.json` (or `web/.well-known/assetlinks.json` for legacy Expo Webpack websites), and collect the following information:
+
+- `package_name`: The Android [application ID](https://docs.expo.dev/versions/latest/config/app/#package) of your app (e.g. `com.bacon.app`). This can be found in the `app.json` file under `expo.android.package`.
```

Comment:
```suggestion
- `package_name`: The Android [application ID](/versions/latest/config/app/#package) of your app (for example, `com.bacon.app`). This can be found in the **app.json** file under `expo.android.package`.
```